infra/gcb/build_base_images.py

```diff
@@ -64,6 +71,7 @@ def main():
       projectId='oss-fuzz-base', body=build_body).execute()
   build_id = build_info['metadata']['build']['id']
 
+  print >> sys.stderr, 'Logs:', get_logs_url(build_id)
```

Comment:
Why print on stderr, why not just print ? same for other one.

Reply:
Just following the existing format from https://github.com/google/oss-fuzz/blob/master/infra/gcb/build.py#L345